OpenRA.Game/Map/Map.cs

```diff
@@ -165,41 +183,65 @@ static object LoadVideos(MiniYaml y)
 			return videos;
 		}
 
-		[FieldLoader.Ignore] public Lazy<Dictionary<string, ActorReference>> Actors;
+		[FieldLoader.Ignore]
```

Comment:
Style nit: Convention is to have these on the same line, so these add unwanted to noise to the commit.